code/modules/modular_computers/file_system/programs/generic/bounty_board.dm

```diff
@@ -0,0 +1,140 @@
+/*
+
+*/
+
+/datum/computer_file/program/bounty_board_app
+	filename = "bountyBoard"
+	filedesc = "Bounty board"
+	extended_desc = "This program allows creating and fulfilling requests."
+	program_icon_state = "generic"
+	program_key_state = "generic_key"
+	size = 2
+	requires_ntnet = 1
+	available_on_ntnet = 1
+	nanomodule_path = /datum/nano_module/bounty_board
+	usage_flags = PROGRAM_ALL
+
+
+/datum/nano_module/bounty_board
+	name = "Bounty board"
+	var/datum/computer_file/report/bounty_entry/selectedEntry
+
+/datum/nano_module/bounty_board/ui_interact(mob/user, ui_key = "main", datum/nanoui/ui = null, force_open = 1, state = GLOB.default_state)
+	var/list/data = host.initial_data()
```

Comment:
Better move the generation of UI data to `ui_data(mob/user)` proc. Necessary for a future UI refactor.